src/chart/layout.js

```diff
@@ -5,7 +5,7 @@
 export function layout() {
     var wrapper = this.wrap
         .append('div')
-        .attr('class', 'aeTable row-fluid')
```

Comment:
@samussiah I don't understand this update, so wanted to draw your attention to it

Reply:
Doesn't look like we need it, although it is a bootstrap class.  Go ahead and remove row-fluid, wc-row-fluid, and form-inline.  They're not referenced anywhere else.

Reply:
@samussiah Sounds good, removed those classes